app/assets/stylesheets/overrides.scss

```diff
@@ -0,0 +1,43 @@
+$brown: #5e412f;
+$cream: #fcebb6;
+$turq: #78c0a8;
+$orange: #f07818;
+$ltorange: #f0a830;
+
+
+$navmain: $ltorange;
+$navactive: $orange;
+$tabdown: $cream;
+$font: $brown;
+
+.five-up.nav-top {
+  background-color: $navmain;
+  padding: 0px;
+}
+
+.five-up dd.active a {
+  background-color: scale-color($navmain, $lightness: -12%);
+}
+
+.five-up.nav-bottom dd a{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space